api/src/main/java/org/openmrs/api/impl/ConceptServiceImpl.java

```diff
@@ -216,27 +234,14 @@ public Concept saveConcept(Concept concept) throws APIException {
 				concept.getSynonyms(locale).iterator().next().setLocalePreferred(true);
 			}
 			//index terms are never used as preferred name
-			
+
 			if (possiblePreferredName != null) { //there may have been none
 				possiblePreferredName.setLocalePreferred(true);
 			}
 			checkedLocales.add(locale);
 		}
-		
-		//See TRUNK-3337 for why we set changed by and date changed every time we save a concept.
-		concept.setDateChanged(new Date());
-		concept.setChangedBy(Context.getAuthenticatedUser());
```

Comment:
Can you do only what the ticket is all about and no more? Not formatting, no nothing else. It greatly simplifies the review. 😊